amethyst_animation/src/systems/control.rs

```diff
@@ -194,6 +194,12 @@ where
             controls.remove(entity);
         }
     }
+
+    fn setup(&mut self, res: &mut Resources) {
+        Self::SystemData::setup(res);
+        ReadStorage::<AnimationSet<I, T>>::setup(res);
+        ReadStorage::<Handle<Animation<T>>>::setup(res);
```

Comment:
Shouldn't this get added by the processor?

Reply:
Already discussed. The Animaton handle should not be used as a component.